variant-remapping-automation/remapping_automation.py

```diff
@@ -69,7 +73,7 @@ def write_remapping_process_props_template(template_file_path):
 spring.data.mongodb.password={mongo_pass}
 
 spring.data.mongodb.authentication-database=admin
-mongodb.read-preference=secondaryPreferred
+mongodb.read-preference=primaryPreferred
```

Comment:
A TODO could be added here to indicate that this needs to be reverted post 4.0 migration.

Reply:
Done